composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "laminas-commons/lmc-rbac",
+  "name": "lm-commons/lmc-rbac",
   "description": "Laminas Module that provides a layer of features of Laminas\\Permissions\\Rbac",
```

Comment:
this module has nothing related to `Laminas\Permissions\Rbac` maybe just the RBAC in the name